Avatar/avatar.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="stylesheet" href="avatar.css" />
+		<title>Avatar</title>
+	</head>
+	<body>
+		<h1>Avatar Component</h1>
+		<img src="../Assets/avatar-1.jpg" alt="" class="avatar avatar-lg" />
```

Comment:
**FEAT:** Give a value to "alt" attribute as it helps for accessibility.

Reply:
thank you Vishal. I will change it right away